assets/css/addPost.css

```
@@ -0,0 +1,109 @@
+.container {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    border: none;
+    box-shadow: none;
+}
+
+
+#addPost-container {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    align-items: center;
+    background-color: var(--feedBackgroundColor);
+    border-radius: 15px;
+    width: 400px;
+    padding: 20px;
+    margin: 40px auto;
+    box-shadow: rgba(0, 0, 0, 0.35) 0px 5px 15px;
+    margin: 40px auto;
+}
+
+.post {
+    background-color: #fff;
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+}
+
+.post-row {
+    display: flex;
+    width: 100%;
+    justify-content: space-between;
+    margin-bottom: 15px;
+}
+
+.post-row textarea,
+.post-row input {
+    flex-grow: 1;
+}
+
+.post-row textarea {
+    font-size: 1rem;
+    font-family: inherit;
+    height: 150px;
+    padding: 10px;
+    border: 1px solid #ccc;
+    border-radius: 5px;
+    font-size: 1rem;
+    resize: none;
+}
+
+.post-row label {
+    flex-grow: 0;
+    margin-right: 10px;
+    align-self: center;
+    width: 100px;
+    text-align: right;
```

```suggestion
    text-align: right;
    font-size: 1.15rem;
```
Make the font size slightly better to distinguish from the actual upload button